transformation_test/testdata/logging_processor-elasticsearch-json-ecs/input.log

```diff
@@ -0,0 +1,6 @@
+{"@timestamp":"2025-09-19T02:20:17.920Z", "log.level": "INFO", "message":"initialized", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"es-test-3","elasticsearch.cluster.name":"elasticsearch"}
```

Comment:
Please add an example of a "multiline json log" to verify it works in ES 8.x. Note : There is a bug in fluent-bit that causes the last line not to be processed in transformation tests when there is a multi-line processor, so to process the multiline log it should no be the last log line.